cmd/root/auth_test.go

```diff
@@ -181,3 +182,54 @@ func TestWorkspaceClientOrPrompt(t *testing.T) {
 		})
 	})
 }
+
+func TestMustAccountClientWorksWithDatabricksCfg(t *testing.T) {
+	testutil.CleanupEnvironment(t)
+
+	dir := t.TempDir()
+	configFile := filepath.Join(dir, ".databrickscfg")
+	err := os.WriteFile(
+		configFile,
+		[]byte(`
+			[account-1111]
+			host = https://accounts.azuredatabricks.net/
+			account_id = 1111
+			token = foobar
+			`),
+		0755)
+	require.NoError(t, err)
+
+	ctx, _ := cmdio.SetupTest(context.Background())
+	cmd := New(ctx)
+
+	t.Setenv("DATABRICKS_CONFIG_FILE", configFile)
+	err = MustAccountClient(cmd, []string{})
+	require.NoError(t, err)
+}
+
+func TestMustAccountClientWorksWithNoDatabricksCfgButEnvironmentVariables(t *testing.T) {
+	testutil.CleanupEnvironment(t)
+
+	ctx := context.Background()
+	cmd := New(context.Background())
+	cmdIO := cmdio.NewIO(flags.OutputText, cmd.InOrStdin(), cmd.OutOrStdout(), cmd.ErrOrStderr(), "")
+	cmd.SetContext(cmdio.InContext(ctx, cmdIO))
```

Comment:
Can this use `ctx, _ := cmdio.SetupTest(context.Background())` like the test above?